testdata/cheats/Parse.t.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity >=0.8.0;
+
+import "ds-test/test.sol";
+import "../logs/console.sol";
+import "./Cheats.sol";
+
+contract ParseTest is DSTest {
+    Cheats constant cheats = Cheats(HEVM_ADDRESS);
+
+    function testParseBytes() public {
+        string memory stringBytes = "7109709ECfa91a80626fF3989D68f67F5b1DD12D";
+        bytes memory testBytes = hex"7109709ECfa91a80626fF3989D68f67F5b1DD12D";
+        assertEq0(testBytes, cheats.parseBytes(stringBytes));
```

Comment:
we don't have `assertEq(bytes, bytes)` declaration, only `assertEq0`. Should probably add it too?